lib/modules/manager/npm/update/locked-dependency/yarn-lock/replace.spec.ts

```diff
@@ -114,10 +114,9 @@ describe('modules/manager/npm/update/locked-dependency/yarn-lock/replace', () =>
         "
       `);
       expect(removedSections[0].value).toMatchInlineSnapshot(`
-        "string-width@^1.0.1:
-          version "1.0.2"
-          resolved "https://registry.yarnpkg.com/string-width/-/string-width-1.0.2.tgz#118bdf5b8cdc51a2a7e70d211e07e2b0b9b107d3"
-          integrity sha1-EYvfW4zcUaKn5w0hHgfisLmxB9M=
+        "  version "2.0.0"
```

Comment:
Is this change correct? Did we have a wrong test before?

Reply:
Seems like so, at least now it's consistent with other tests in this file